### PR TITLE
Link from coding guide to tech radar

### PR DESCRIPTION
**Pull request by jacobe**
... and add missing CMS technologies to the radar

Related to #64

### CODE REVIEW

File: docs/guides/coding-guide.md [hunk 1]
```diff
@@ -72,24 +72,9 @@ Offers better performance and easier code generation, but a bit harder to set up
...
-
-### 3.2 Back-end
-* **.Net 6+** is the preferred framework for developing web APIs. We also have projects using **Node JS** and **Go**.
-
-### 3.3 App Development
-* **Flutter** is the preferred framework for developing native apps
-
-### 4.4 Content Management
-* **WordPress** is the preferred content management system for building websites.
-* **Directus** is currently the preferred content management system for applications
```
Comment (laurensgroeneveld) [inline docs/guides/coding-guide.md:92]: I feel like the information written down here doesn't directly translate to the technology radar, for example the categorization with frontend/backend/cms. I think the tech radar could complement this but not replace it (for example, things like logo assets or the upcoming Vue component library have no place in the tech radar, which is strictly for how we view technologies).

Comment (jacobe) [reply docs/guides/coding-guide.md:92]: Agreed, it makes sense. Let me take another round with this. The list here could include tech in "adopt" only, for example, detailing the standards more than the tech radar does.

File: docs/guides/tech-radar.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ If anything is not on the radar it means that we have never considered it seriou
 
 ## How To Add
 
-Edit <./data.js> and add your technology to the `technologies` array.
+Edit [tech-radar.js](/.vuepress/components/tech-radar.js) and add your technology to the `technologies` array.
```
Comment (laurensgroeneveld) [inline docs/guides/tech-radar.md:19]: I don't think this link will work, as only converted Markdown files get deployed (as HTML). Previously I've just added an absolute link to the file in the repository (https://github.com/bcc-code/bcc-code.github.io/blob/master/.vuepress/components/tech-radar.js)

**Review by laurensgroeneveld — APPROVED** [review]
Just a small comment on the link, looks good otherwise 👍

**Pull request merged**